examples/neural_networks/plot_rbm_logistic_classification.py

```diff
@@ -117,8 +125,9 @@ def shift(x, w):
     % (metrics.classification_report(Y_test, Y_pred))
 )
```

Comment:
It might be nice to split this cell so that we don't need to scroll through the output to look at both classification reports.

Reply:
Thank you for your comments! Should I make these changes as a new PR?

Reply:
I have tackled @jsilke's comment in one of my commits you can see the rendered example in the development doc [here](https://scikit-learn.org/dev/auto_examples/neural_networks/plot_rbm_logistic_classification.html#evaluation)